pkg/proc/eval.go

```diff
@@ -1790,7 +1790,7 @@ func (v *Variable) findMethod(mname string) (*Variable, error) {
 	}
 
 	if _, istypedef := typ.(*godwarf.TypedefType); !istypedef {
-		return nil, nil
+		return v.tryFindMethodInEmbeddedFileds(mname)
```

Comment:
This is because 1.10 doesn't support function calls and will accept everything and versions of Go prior to 1.12 have a linker bug that's too complicated to explain here (if you are curious look at commit a0f5d5f8830e578892c47f7704e6a2616273aac1 in the go repository). I think you should keep `return nil, nil` here and only test this feature on 1.12 and later (which, if you do the change I suggest below simply means adding the test cases to the testcases112 variable).